services/core-api/app/api/mines/permits/permit_conditions/models/standard_permit_conditions.py

```diff
@@ -1,19 +1,18 @@
-from datetime import datetime
-
-import uuid
-
+from typing import Sequence
 from sqlalchemy.dialects.postgresql import UUID
-from sqlalchemy.orm import validates, backref
-from app.extensions import db
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import backref
 from sqlalchemy.schema import FetchedValue
 
+from app.extensions import db
 from app.api.utils.models_mixins import SoftDeleteMixin, AuditMixin, Base
+from app.api.utils.list_lettering_helpers import num_to_letter, num_to_roman
 
 
 class StandardPermitConditions(SoftDeleteMixin, AuditMixin, Base):
     __tablename__ = 'standard_permit_conditions'
 
-    standard_permit_condition_id = db.Column(db.Integer, primary_key=True)
+    standard_permit_condition_id = db.Column(db.Integer, primary_key=True,server_default=FetchedValue())
```

Comment:
Looks like the formatter needs to run in this file. There should be a space after the ",".